### DISCUSSION TITLE
Antmedia is throwing "highResourceUsage" error when more than 10 streams in the server.

### DISCUSSION THREAD

**Discussion by MBFLYT** [ant-media/Ant-Media-Server#5910] [Q&A]
@USAMAWIZARD @lastpeony 

Antmedia is throwing  {   “definition”: “highResourceUsage”,  “command”: “error”} when the stream count is more than 10 in the Antmedia server, can you guys please suggest what we should do to resolve this?

**Comment by USAMAWIZARD** [comment]
@MBFLYT are you using any adaptive bitrate on the server side ?? 
how many core server are you using ? usually this error is cased because of high ram usage or CPU usage.

**Reply by sogusogu** [reply]
@USAMAWIZARD we are using a 4GBram 2 Core machine

no adaptive bitrate enabled as of now

**Reply by USAMAWIZARD** [reply]
Hi @sogusogu  minimum recommended version to run Ant Media Server is 4 core server. Can you please update CPU of the server to  4 core. I think that is expected behavior because server is running out of resources.
